plugin/pom.xml

```diff
@@ -46,6 +46,11 @@
   </developers>
 
   <dependencies>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>caffeine-api</artifactId>
+      <version>2.9.0-17.v79e9099a51e3</version>
```

Comment:
```suggestion
      <version>2.9.1-23.v51c4e2c879c8</version>
```

Reply:
just to prevent a needless change from DependaBot https://github.com/jenkinsci/caffeine-api-plugin/releases